nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/core/ElasticSearchEmbeddedNode.java

```diff
@@ -133,12 +134,13 @@ public void close() throws IOException {
     }
 
     protected void deleteLuceneFileLock(String root) throws IOException {
-        List<Path> locks = Files.walk(Paths.get(root))
-                                .filter(f -> f.getFileName().toString().equals("node.lock"))
-                                .collect(Collectors.toList());
-        if (!locks.isEmpty()) {
-            locks.forEach(f -> log.warn("Found lock on close, deleting: " + f));
-            locks.forEach(f -> f.toFile().delete());
+        try (Stream<Path> files = Files.walk(Paths.get(root))) {
+            List<Path> locks = files.filter(f -> f.getFileName().toString().equals("node.lock"))
+                                    .collect(Collectors.toList());
+            if (!locks.isEmpty()) {
+                locks.forEach(f -> log.warn("Found lock on close, deleting: " + f));
+                locks.forEach(f -> f.toFile().delete());
```

Comment:
Here you are using `aFile.delete()`,  yesterday you changed similar code to `Files.delete(aFile.toPath())`.

Reply:
Here I'm just keeping the current code. I've not done a global refactor of File-based operations to Files+Path-based ones (although I may have changed a few bugging me in the past days yes).